From 43d059863f6ef181ae871d23cf0d78fd100118a7 Mon Sep 17 00:00:00 2001
From: srinidhira0 <srinidhi.rao@broadcom.com>
Date: Wed, 25 Jun 2025 11:16:39 +0000
Subject: [PATCH] jitterentropy: Add prototype for sample collection

  -  Add the function prototypes for functions that collect
     the jitterentropy samples.

Signed-off-by: srinidhira0 <srinidhi.rao@broadcom.com>
---
 include/crypto/internal/rng.h | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/include/crypto/internal/rng.h b/include/crypto/internal/rng.h
index e0711b6a5..fee78b2bd 100644
--- a/include/crypto/internal/rng.h
+++ b/include/crypto/internal/rng.h
@@ -37,4 +37,9 @@ static inline void crypto_rng_set_entropy(struct crypto_rng *tfm,
 	crypto_rng_alg(tfm)->set_ent(tfm, data, len);
 }
 
+int jitterentropy_kcapi_write_sample_file(const char *sample_file_name,
+						unsigned long rounds);
+int jitterentropy_kcapi_write_restart_files(const char *sample_file_name,
+						unsigned long rounds);
+
 #endif
-- 
2.35.6

